src/multi_library_class_loader.cpp

```diff
@@ -88,13 +86,14 @@ void MultiLibraryClassLoader::shutdownAllClassLoaders()
 int MultiLibraryClassLoader::unloadLibrary(const std::string& library_path)
 {
   int remaining_unloads = 0;
-  if(isLibraryAvailable(library_path))
+  LibraryToClassLoaderMap::iterator itr = active_class_loaders_.find(library_path);
+  if (itr != active_class_loaders_.end())
```

Comment:
Because I do need the `itr` to erase it in line 96.
This way, we avoid finding the library several times in a row.